psydac/feec/multipatch/examples_nc/hcurl_eigen_pbms_dg.py

```diff
@@ -0,0 +1,323 @@
+"""
+    Solve the eigenvalue problem for the curl-curl operator in 2D with DG discretization
+"""
```

Comment:
which DG scheme is used here ? A reference would be useful.

Also, same question as above: is this example tested in the CI?

Reply:
Do you have a reference in mind? I think it's based on a implementation from Said + some lecture notes of mine. 

See `psydac/feec/multipatch/tests/test_feec_maxwell_multipatch_2d.py`.

Reply:
Apparently I was referring to: A. Buffa and I. Perugia, “Discontinuous Galerkin Approximation of the Maxwell Eigenproblem”, 2006.